tests/fixtures/couplet.json

```diff
@@ -147,14 +147,12 @@
     "rhythm": {
       "stress": "---+--+--+-",
       "type": "pattern",
-      "length": 11
-    },
-    "structure": "couplet",
-    "rhyme": "a",
-    "ending": "eda",
-    "ending_stress": -3,
-    "rhyme_type": "consonant",
-    "rhyme_relaxation": true
+      "length": 11,
+      "length_range": {
+        "min_length": 11,
+        "max_length": 11
+      }
+    }
```

Comment:
This should not happen as the output in that case should've deleted the `length_range` key, right?

Reply:
I forgot to change the fixtures to reflect the new changes